examples/react/with-provider/src/App.tsx

```diff
@@ -2,12 +2,12 @@ import React from 'react'
 import logo from './logo.svg'
 import './App.css'
 import {
-    useIsDVCInitialized,
-    withDVCProvider,
+    useIsDevCycleInitialized,
+    withDevCycleProvider,
 } from '@devcycle/devcycle-react-sdk'
 import DevCycleExample from './DevCycleExample'
 
-const SDK_KEY = process.env.NX_CLIENT_KEY || '<YOUR_DVC_CLIENT_SDK_KEY>'
+const SDK_KEY = process.env.NX_CLIENT_KEY || '<YOUR_DEVCYCLE_CLIENT_SDK_KEY>'
```

Comment:
should the env vars we use for reading keys in example apps also be updated as part of this naming stuff?

Reply:
I have a separate ticket for that, but yea it should.